From ca905a639ea827e74288a3d49ccf74e2692e2668 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonas=20=C3=85dahl?= <jadahl@gmail.com>
Date: Fri, 25 Feb 2022 19:01:53 +0100
Subject: [PATCH 09/13] sharing/remote-desktop: Hook up to explicit enable
 setting

Enabling the service isn't enough, one also needs to enable the relevant
backend.
---
 panels/sharing/cc-sharing-panel.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/panels/sharing/cc-sharing-panel.c b/panels/sharing/cc-sharing-panel.c
index 3846a6da1..3dd8431e6 100644
--- a/panels/sharing/cc-sharing-panel.c
+++ b/panels/sharing/cc-sharing-panel.c
@@ -1389,6 +1389,11 @@ cc_sharing_panel_setup_remote_desktop_dialog (CcSharingPanel *self)
 
   rdp_settings = g_settings_new (GNOME_REMOTE_DESKTOP_RDP_SCHEMA_ID);
 
+  g_settings_bind (rdp_settings,
+                   "enable",
+                   self->remote_desktop_switch,
+                   "active",
+                   G_SETTINGS_BIND_DEFAULT);
   g_settings_bind (rdp_settings,
                    "view-only",
                    self->remote_control_switch,
-- 
2.34.1

